src/transactions/TransactionUtils.h

```diff
@@ -4,6 +4,7 @@
 // under the Apache License, Version 2.0. See the COPYING file at the root
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
 
+#include "main/Config.h"
```

Comment:
Nit: forward declare config instead of include.

Reply:
Done.